build.sbt

```diff
@@ -326,6 +321,11 @@ def regexFileFilter(s: String): FileFilter = new FileFilter {
   def accept(f: File) = pat.matcher(f.getAbsolutePath.replace('\\', '/')).matches()
 }
 
+// This project provides the STARR scalaInstance for bootstrapping
+lazy val bootstrap = (project in file("target/bootstrap")).settings(
```

Comment:
i wonder about the various ways to create `File`s that we see in `build.sbt`:

  - `file("target/...")`
  - `file(".") / "target" / ...`
  - `(baseDirectory in ThisBuild).value / "target" / ...`
  - `buildDirectory.value / "quick" / ...`

does it matter, should we unify / clean up a bit?